src/notification/email/email.service.ts

```diff
@@ -0,0 +1,27 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import * as sendgrid from '@sendgrid/mail';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class EmailService {
+  readonly #emailAddress: string;
```

Comment:
Ah, I love how I remember stuff later on :D. We can add a logger to this service so we can later track it easier what went wrong.